Install both a .pc file (for pkgconfig) and liblas-config (for autotools)
based builds.
--- apps/CMakeLists.txt.orig	2021-08-28 20:18:25.000000000 -0400
+++ apps/CMakeLists.txt	2021-08-28 20:18:32.000000000 -0400
@@ -216,8 +216,6 @@
       DESTINATION ${LIBLAS_LIB_DIR}/pkgconfig
       PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)

-  else()
-
     # Autoconf compatibility variables to use the same script source.
     set(prefix ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix ${CMAKE_INSTALL_PREFIX}/bin)
